cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/resources/ConnectorNamespaceProvisioner.java

```diff
@@ -203,6 +210,76 @@ private void createResourceQuota(String uow, ConnectorNamespace connectorNamespa
             .createOrReplace(quota);
     }
 
+    private void createResourceLimit(String uow, ConnectorNamespace connectorNamespace) {
+        if (connectorNamespace.getQuota() == null) {
+            return;
+        }
+
+        LimitRangeItem limit = new LimitRangeItem();
+        limit.setType(LIMITS_TYPE_CONTAINER);
+        limit.setDefault(new TreeMap<>());
+        limit.setDefaultRequest(new TreeMap<>());
+
+        if (config.quota().defaultLimits() != null) {
```

Comment:
The code above computes the limits based on the number of connectors and the limits set on the namespace. 

However there may be the case where a connector declares requests that could be above the limits this method would compute, however, since `LimitRange` kicks in only in case of missing requests/limits, a connector could set higher limits.

@rinaldodev what do you think?